### ISSUE TITLE
Jet Engine - Checkbox/Select Form Field for RELATED POSTS!

### ISSUE THREAD

**Issue by Willitbang** [Crocoblock/suggestions#1450]
Hello,
Can you please add Related Posts into the Checkbox/Select form field.
Currently I can select 'Posts' on the checkbox field but this pulls through Every Single Post in this CPT.  Adding this related posts functionality would essentially hide all of the non related posts!

For example, I have 3 CPT's - Matches, Teams, Athletes
- Each Match has 2 Related Teams
- Then the Teams would need to select the Athletes for this Match (this would require a checkbox for athletes related to the current team).

In another example I have Owners and Pets and require the same checkbox functionality as described above (see images attached). 

This functionality seems like it is essential to expanding the capabilities of related posts. This seems quite strait forward could you please add this to the next update.

Many thanks

![Screenshot (2)](https://user-images.githubusercontent.com/66962693/84682269-b1d6e700-af35-11ea-938d-dbc1e6e2b35b.png)
![Screenshot (4)](https://user-images.githubusercontent.com/66962693/84682271-b26f7d80-af35-11ea-9950-4fd3da725ce2.png)

**Comment by oceandiveloper** [comment]
yes please that would be great. the more the capabilities of relationships are expanded within JetEngine, the more complex and therefore for clients valuable applications can be achieved.

**Comment by aalexandr21** [comment]
Please, check this https://jetformbuilder.com/blog/update-jetformbuilder-fields-dynamically/#get-values-list-from-custom-query
